File: aat-lib/src/main/java/ch/bailu/aat_lib/view/graph/PlotterConfig.java
package ch.bailu.aat_lib.view.graph;

import ch.bailu.aat_lib.gpx.GpxList;

public interface PlotterConfig {
    int getWidth();
    int getHeight();

    GpxList getList();
    int getIndex();

    boolean isXLabelVisible();

    LabelInterface getLabels();
}

File: aat-lib/src/main/java/ch/bailu/aat_lib/view/graph/PlotterConfig.kt
package ch.bailu.aat_lib.view.graph

import ch.bailu.aat_lib.gpx.GpxList

interface PlotterConfig {
    fun getWidth(): Int
    fun getHeight(): Int
    fun getList(): GpxList
    fun getIndex(): Int
    fun isXLabelVisible(): Boolean
    fun getLabels(): LabelInterface
}
